tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "compilerOptions": {
     "declaration": true,
     "rootDir": "src",
-    "outDir": "dist"
+    "outDir": "dist",
+    "esModuleInterop": true
```

Comment:
why was this necessary?

Reply:
Hi @kibertoad!

Due to the usage of both "import" and "require" on the test `fastify-swagger.spec.ts`, the following warning pops up multiple times:

```
ts-jest[config] (WARN) message TS151001: If you have issues related to imports, you should consider setting `esModuleInterop` to `true` in your TypeScript configuration file (usually `tsconfig.json`). See https://blogs.msdn.microsoft.com/typescript/2018/01/31/announcing-typescript-2-7/#easier-ecmascript-module-interoperability for more information.
```

I just followed the advice and added the required property from tsconfig.json. Another way to solve this would be changing the "require" to an import but I didn't dig into that.

Reply:
is this still needed after the import change?